canonical-web-and-design/snapcraft.io
# Add Snapcrafters publisher page

## Done
Created a publisher page for Snapcrafters 

## QA
- Go to https://snapcraft-io-3796.demos.haus/publisher/snapcrafters
- Compare to the other publisher pages:
  - https://snapcraft-io-3796.demos.haus/publisher/jetbrains
  - https://snapcraft-io-3796.demos.haus/publisher/kde
- Check it looks as it should

## Issue
Fixes [#2267](https://github.com/canonical-web-and-design/marketplace-tribe/issues/2267)
